app/views/forms/edit/document-selector.js

```diff
@@ -429,7 +429,7 @@ function ($timeout, locale, $filter, $q, searchService, appConfigService, IStora
 
                 var queryParameters = {
                     fields       : ($attr.displayFields||'')!= '' ? $attr.displayFields : undefined,
-                    'query'      : queryField + ':("' + identifiers.join('" "') +'")',
+                    'query'      : queryField + ':("' + solr.escape(identifiers.join('" "')) +'")',
```

Comment:
same here escape then join
```suggestion
                    'query'      : queryField + ':("' +_.map(identifiers,solr.escape).join('" "') +'")',
```